profile/README.md

```diff
@@ -8,99 +8,24 @@
 
 ***
 
-## Overview
 <details><summary>Why Gno?</summary>
 
-- A better programming language for smart contracts: Gno.land utilizes the Gno programming language, which is an interpreted variation of Go. It is executed through a specialized virtual machine called the GnoVM, purpose-built for the Gno.land blockchain. There are some differences among the languages: Gno does not support Goroutines, channels and non-deterministic libraries like `crypto/rand`, and it supports additional features like automatic persistence of global variables for storing state.
+gno.land addresses the need for decentralized social platforms that enhance transparency and user autonomy. Unlike centralized platforms that leave users powerless, gno.land empowers diverse voices, enabling meaningful discourse without censorship.
```

Comment:
The question is "Why Gno?" and then the first thing you talk about it gno.land instead of Gno. Maybe rename to "Why Gno & gno.land?"

Reply:
Gno.land addresses the social network; Gnoland addresses the technical thing.

 Why Gno is more general, more about the project. I think it’s fine. Maybe @thehowl who also worked on the topic can help finding an ideal phrasing.

Reply:
We talk about Gno in the following paragraph, it's fine.

Like for the `Why Gno?` document, I think it's OK because 1. it talks about the project as a whole 2. focusing on the social part first is good, because the "social problem" justifies why we need Gno (as a programming language).